examples/realtime/lslclient_rt_plot.py

```diff
@@ -0,0 +1,48 @@
+"""
+==============================================================
+Plot real-time epoch data with LSL client
+==============================================================
+
+This example demonstrates how to use the LSL client to plot real-time
+collection of event data from an LSL stream.
+For the purposes of demo, a mock LSL stream is constructed. You can
+replace this with the stream of your choice by changing the host id to
+the desired stream.
+
+"""
+# Author: Teon Brooks <teon.brooks@gmail.com>
+#
+# License: BSD (3-clause)
+import matplotlib.pyplot as plt
+
+from mne.realtime import LSLClient, MockLSLStream
+
+print(__doc__)
+
+# this is the host id that identifies your stream on LSL
+host = 'mne_stream'
+# this is the max wait time in seconds until client connection
+wait_max = 5
+
+
+# For this example, let's use the mock LSL stream.
+stream = MockLSLStream(host)
```

Comment:
could this be made to work for a certain file? Like the MNE sample data? instead of a random number

Reply:
closing for thread below